### DISCUSSION TITLE
"C shape"blocks

### DISCUSSION THREAD

**Discussion by MA-master-sec** [TurboWarp/extensions#366] [Ideas]
I found some 'special' code
like `blockType:Scratch.BlockType.CONDITIONAL,`
and `util.startBranch(1, false);`

However,it have something wrong...

**Comment by GarboMuffin** [comment]
they are not documented and are incompatible with the compiler

**Reply by OeildeLynx31** [reply]
If I don't have the compiler, does it work ?

**Reply by MA-master-sec** [reply]
yes,it dose.
Although you have he compiler,it will work,too
however,it'll 
![image](https://user-images.githubusercontent.com/126303227/235881541-9c0dcb67-4f56-4f85-b464-247b4c1aa87c.png)
(Maybe it's my wrong)

**Reply by OeildeLynx31** [reply]
Thank you 👍

**Comment by MA-master-sec** [comment]
It worked
![image](https://user-images.githubusercontent.com/126303227/235880805-5abe0e94-8930-4845-b5c4-762d50f58439.png)
![image](https://user-images.githubusercontent.com/126303227/235880952-ef1d0e09-4ff5-4399-a543-8709651f2dfb.png)

**Reply by Sunny-da-moth** [reply]
theres also LOOP which is used in forever and repeat and repeat until blocks which have the little arrow icon on the bottom and have additional util.stackFrame.loopCounter

**Reply by MA-master-sec** [reply]
I had found it
but it throw an error
